openwisp_monitoring/monitoring/settings.py

```diff
@@ -2,3 +2,11 @@
 
 ADDITIONAL_CHARTS = getattr(settings, 'OPENWISP_MONITORING_CHARTS', {})
 ADDITIONAL_METRICS = getattr(settings, 'OPENWISP_MONITORING_METRICS', {})
+
+RETRY_OPTIONS = getattr(
+    settings,
+    'OPENWISP_MONITORING_RETRY_OPTIONS',
```

Comment:
I think this should be called `OPENWISP_MONITORING_WRITE_RETRY_OPTIONS`, otherwise it can be misleading